src/parser/shaman/enhancement/CHANGELOG.tsx

```diff
@@ -6,6 +6,7 @@ import SpellLink from 'common/SpellLink';
 import { change, date } from 'common/changelog';
 
 export default [
+  change(date(2020, 4, 25), <>Converted remaining Enhancement Shaman files to typescript.</>, [Vetyst]),
```

Comment:
Super minor, but its may not april. 5,25

Reply:
You're right 😄